iris_grib/_load_convert.py

```diff
@@ -1195,6 +1248,9 @@ def grid_definition_section(section, metadata):
     elif template == 5:
         # Process variable resolution rotated latitude/longitude.
         grid_definition_template_5(section, metadata)
+    elif template == 10:
+        # Process Mercator
```

Comment:
\<pedant>Full stop at the end of the comment, please.\</pendant>

Reply:
Frankly, the word "process" is superfluous too. 😉